packages/usa-form/src/test/test-pattern/test-usa-form.twig

```diff
@@ -1,10 +1,11 @@
 <form class="usa-form">
   <div class="usa-character-count">
-    <div class="usa-form-group">
-      <label class="usa-label" for="with-hint-input">Character count</label>
+    <div class="usa-form-group {% if error_state %}usa-form-group--error{% endif %}">
+      <label class="usa-label {% if error_state %}usa-label--error{% endif %}" for="with-hint-input">Character count{% if error_state %} has an error{% endif %}</label>
```

Comment:
The spacing for `has an error` looks inconsistent.

Reply:
Resolved in [dc0cfbc](https://github.com/uswds/uswds/pull/5879/commits/dc0cfbc6f482ea079817ddfa5f262ccecb3a6ce9)